storage_visualization/disk_usage.py

```diff
@@ -56,6 +56,8 @@ def aggregate_level(name: str) -> str:
         return name[: mt_index + 3]
     if (vds_index := name.find('.vds/')) != -1:
         return name[: vds_index + 4]
+    if (batch_tmp_index := name.rfind('batch-tmp/')) != -1:
```

Comment:
All the manky stuff that's recently appeared in cpg-*-test-tmp actually matches `/batch-tmpNNNNNN/` where the NNNNNN appears to be six hex digits. So alas this won't catch it.

(And we should figure out what's not following the pattern!)